### DISCUSSION TITLE
change order of event

### DISCUSSION THREAD

**Discussion by chlgbdud** [nhn/tui.calendar#1285] [Q&A] [answered]
![스크린샷 2022-09-21 오후 12 56 29](https://user-images.githubusercontent.com/95602981/191411079-6ddfb8bc-2fb7-4bd9-b1fc-669a60b35805.png)

i want change order of event.

group that have same start time and "end time" is sorted correct.
but i want sort only start time. what can i do?

**Comment by adhrinae** [comment, marked as answer]
Unfortunately, we don't offer a way to set the order of events.

**Reply by chlgbdud** [reply]
ty
